src/ResourceManagement/RecoveryServices/RecoveryServicesManagement/Generated/Models/VaultUsageListResponse.cs

```diff
@@ -0,0 +1,71 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Hyak.Common;
+using Microsoft.Azure;
+using Microsoft.Azure.Management.RecoveryServices.Models;
+
+namespace Microsoft.Azure.Management.RecoveryServices.Models
+{
+    /// <summary>
+    /// The response model for Vault usage.
+    /// </summary>
+    public partial class VaultUsageListResponse : AzureOperationResponse, IEnumerable<Usage>
```

Comment:
shouldn't the types by lowerCamelcased? like name instead of Name, currentValue instead of currentValue etc? Not sure if Ibiza is expecting this contract?